.github/workflows/firebase-hosting-merge-prod.yml

```diff
@@ -0,0 +1,30 @@
+# This file was auto-generated by the Firebase CLI
```

Comment:
do we need to add the full file here? why can't we just add the main branch to the `branches` tag in `firebase-hosting-merge.yml`?

Reply:
We're deploying to different environments here and there. Here, we're deploying to prod, there we're deploying to staging

Reply:
ah got it